ansible/roles/ocp-workload-migration/templates/operator.yml.j2

```diff
@@ -0,0 +1,245 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    openshift.io/node-selector: ""
+  labels:
+    control-plane: controller-manager
+    controller-tools.k8s.io: "1.0"
+  name: "openshift-migration"
```

Comment:
@djwhatle mig_migration_namespace

Reply:
Fixed

Reply:
Should this namespace name use the `mig_migration_namespace` variable?

Reply:
Did a bulk find/replace and missed this one since it starts with `name` instead of `namespace`. Nice catch @jkupferer